docs/cash-savings/Interest.md

```diff
@@ -0,0 +1,31 @@
+---
+tags: [Cash Savings]
+---
+
+# Interest
+
+## How interest is paid
+
+Interest is paid on the 1st of each month on an accrual basis. This means that interest is calculated (accrued) daily, but paid monthly. 
+
+## Interest rates
+
+Interest rates can be configured per product, split between the investor and the tenant, up to a maximum of the current [Bank of England base rate](https://www.bankofengland.co.uk/monetary-policy/the-interest-rate-bank-rate).
+
+## Rate changes
+
+Sometimes the Bank of England changes the base rate. When this happens we will use the most up to date figure and calculate the accrual for that day onwards accordingly. 
```

Comment:
This bit is hard to be specific on, as I've no idea what are plans are, e.g. when the rate goes down, do we decrease the tenant share, investor share, neither, both, it depends? Same when the rate goes up.

Reply:
We should just say that rates will change inline with BoE base rate, I think that's how Griffin word it on their docs.